manifest.go

```diff
@@ -144,19 +144,19 @@ func toPossible(pp gps.ProjectProperties) (p possibleProps) {
 		case gps.IsSemver, gps.IsVersion:
 			p.Version = v.String()
 		}
-	} else {
-		// We simply don't allow for a case where the user could directly
-		// express a 'none' constraint, so we can ignore it here. We also ignore
-		// the 'any' case, because that's the other possibility, and it's what
-		// we interpret not having any constraint expressions at all to mean.
-		//if !gps.IsAny(pp.Constraint) && !gps.IsNone(pp.Constraint) {
-		if !gps.IsAny(pp.Constraint) && pp.Constraint != nil {
-			// Has to be a semver range.
-			p.Version = pp.Constraint.String()
-		}
+		return p
```

Comment:
Good point. Removed the naked return here. Naming the return value is not adding the self-documentation of this function. The signature with the type is already self documentary enough.

Don't ever use naked return other than self-documentation reasons.

Even though it is not a hard rule, more people tend to return with actual values to avoid nasty bugs. It is hard to keep up with the latest state of a variable if a block has many branches and keep mutating a variable.

So, avoid naked returns, return as soon as possible, prefer to repeat the variables on the return statement.